### PR TITLE
Attack enemy

### PR DESCRIPTION
**Pull request by matharotheelf**

### CODE REVIEW

**Review by Sindex42 — APPROVED** [review]

File: src/hero.py [hunk 1]
```diff
@@ -17,9 +17,11 @@ def __init__(self, game, x_pos, y_pos):
         self.game = game
         self.x_pos = x_pos
         self.y_pos = y_pos
-        self.link_animation_setup()
+        self.orthogonal_boy_animation_setup()
```
Comment (michaelnguyen974) [inline src/hero.py]: Orthogonal boy!

File: src/main.py [hunk 2]
```diff
@@ -106,14 +107,30 @@ def events(self):
                 self.playing = False
 
             if event.type == pg.KEYDOWN:
-                if event.key == pg.K_LEFT:
+                if event.key == pg.K_a:
                     self.hero.move(d_x=-1)
-                if event.key == pg.K_RIGHT:
+                if event.key == pg.K_d:
```
Comment (michaelnguyen974) [inline src/main.py:112]: Will need to extract this :)

**Review by hotblack86 — APPROVED** [review]
Awesome. Hitbox module is very clean. Pie boy's quest begins!

**Review by CKKH — APPROVED** [review]
I approve

Event class to be extracted

**Pull request merged**